### PR TITLE
Add arm doe portal details

### PR DESCRIPTION
**Pull request by mgrover1**
This is required for using ARM data

### CODE REVIEW

File: .github/workflows/build-book.yaml [hunk 1]
```diff
@@ -61,6 +72,9 @@ jobs:
         run: mamba env update -n ${{ inputs.environment_name }} -f ${{ inputs.environment_file }}
 
       - name: Build the book
+        with:
+          ARM_USERNAME: ${{ secrets.ARM_USERNAME }}
+          ARM_PASSWORD: ${{ secrets.ARM_USERNAME }}
```
Comment (brian-rose) [inline .github/workflows/build-book.yaml:77]: Should this be `${{ secrets.ARM_PASSWORD }}`?

Comment (mgrover1) [reply .github/workflows/build-book.yaml:77]: Yes... good catch!

**Comment by mgrover1** [comment]
@brian-rose I made those changes!

**Review by brian-rose — APPROVED** [review]
Looks good!

**Pull request merged**